src/In.ProjectEKA.HipService/Logger/Log.cs

```diff
@@ -0,0 +1,38 @@
+
+namespace In.ProjectEKA.HipService.Logger
+{
+    using System;
+    
+    public static class Log
+    {
+        public static void Information(string format, params object[] arg)
+        {
+            Serilog.Log.Information(format, arg);
+        }
```

Comment:
Can't the format be predefined as everything is custom here now?

Reply:
format is defined in the template file